Topic5Mysql/src/DataBaseTest.java

```diff
@@ -0,0 +1,74 @@
+import java.net.ConnectException;
+import java.sql.ResultSet;
+
+public class DataBaseTest {
+
+	public static void main(String[] args) throws Exception {
+
+		// Connection
+
+		DataBaseConnection dataBaseConnection = new DataBaseConnection();
+		dataBaseConnection.connect();
+
+		// Test exercise 4
+
+		System.out.println(" ");
+		System.out.println(" ---Excercise 4--- ");
+		System.out.println(" ");
+
+		DataBaseManager dataBaseManager = new DataBaseManager();
+
+		ResultSet resultSet1;
+		ResultSet resultSet2;
+
+		int course_id = 2;
+
+		String query1 = "SELECT course.course_name, teacher.last_name, teacher.first_name " + "FROM course, teacher " + "WHERE course_id ="

```

Comment:
To have a better understanding on SQL Queries please look this Visual Representation of SQL joins -> http://www.codeproject.com/Articles/33052/Visual-Representation-of-SQL-Joins